src/Microsoft.Health.Fhir.Core/Exceptions/CustomerManagedKeyException.cs

```diff
@@ -7,14 +7,14 @@
 
 namespace Microsoft.Health.Fhir.Core.Exceptions
 {
-    public class CustomerManagedKeyInaccessibleException : FhirException
+    public class CustomerManagedKeyException : FhirException
     {
-        public CustomerManagedKeyInaccessibleException()
+        public CustomerManagedKeyException(string message)
         {
             Issues.Add(new OperationOutcomeIssue(
                     OperationOutcomeConstants.IssueSeverity.Error,
                     OperationOutcomeConstants.IssueType.Forbidden,
-                    Resources.CustomerManagedKeyInaccessible));
```

Comment:
Should we call the base constructor here so that the `message` gets logged?

Reply:
That sounds good, thanks.  I've made the change.